src/main/java/appeng/decorative/solid/BlockChargedQuartzOre.java

```diff
@@ -49,9 +50,10 @@ public BlockChargedQuartzOre()
 	@Override
 	public Item getItemDropped( final IBlockState state, final Random rand, final int fortune )
 	{
-		for( final Item charged : AEApi.instance().definitions().materials().certusQuartzCrystalCharged().maybeItem().asSet() )
+		Optional<Item> item = AEApi.instance().definitions().materials().certusQuartzCrystalCharged().maybeItem();
+		if( item.isPresent() )
 		{
-			return charged;
+			return item.get();
```

Comment:
This could also use 

```
final Optional<Item> maybeItem = AEApi.instance().definitions().materials().certusQuartzCrystalCharged().maybeItem();

return maybeItem.orElseThrow( () -> new MissingDefinition( "Tried to access charged certus quartz crystal, even though they are disabled" ) );
```